doc/source/cluster/kubernetes/examples/verl-post-training.md

```diff
@@ -0,0 +1,150 @@
+ actor_rollout_ref.actor.optim.lr=1e-6 \
+ actor_rollout_ref.actor.ppo_mini_batch_size=64 \
+ actor_rollout_ref.actor.ppo_micro_batch_size_per_gpu=4 \
+ actor_rollout_ref.rollout.log_prob_micro_batch_size_per_gpu=8 \
+ actor_rollout_ref.rollout.tensor_model_parallel_size=1 \
+ actor_rollout_ref.rollout.gpu_memory_utilization=0.4 \
+ actor_rollout_ref.ref.log_prob_micro_batch_size_per_gpu=4 \
+ critic.optim.lr=1e-5 \
+ critic.model.path=Qwen/Qwen2.5-0.5B-Instruct \
+ critic.ppo_micro_batch_size_per_gpu=4 \
+ algorithm.kl_ctrl.kl_coef=0.001 \
+ trainer.logger=['console'] \
+ trainer.val_before_train=False \
+ trainer.default_hdfs_dir=null \
+ trainer.n_gpus_per_node=4 \
+ trainer.nnodes=1 \
+ trainer.save_freq=-1 \
+ trainer.test_freq=10 \
+ trainer.total_epochs=15 2>&1 | tee verl_demo.log
+```
+
+This job takes hours to complete. While it's running, you can check the Ray dashboard to see more details about the PPO job and the Ray cluster.
+
+```sh
+# Port forward the Ray dashboard to your local machine's port 8265.
+kubectl port-forward verl-cluster-head 8265:8265
+```
+
+Open `127.0.0.1:8265` in your browser to view the Ray dashboard and check whether all GPUs are in use.
+
+![Ray dashboard](../images/verl-ray-dashboard.png)
+
+# Step 7: Check the PPO job logs
+
+Check `verl_demo.log` in the head Pod to see the PPO job's logs.
+For every 10 steps, verl validates the model with a simple math problem.
+
+* Math problem:
+  ```
+  Janet’s ducks lay 16 eggs per day. She eats three for breakfast every morning and bakes muffins for her friends every day with four. She sells the remainder at the farmers' market daily for $2 per fresh duck egg. How much in dollars does she make every day at the farmers' market? Let's think step by step and output the final answer after
+  ```
+* Answer: `(16 - 3 - 4) * 2 = 18`
+
+You should be able to see the model becomes gradually better at this question after several steps.
+
+In this example run, the model first got the correct answer after 130 steps, and the following is the log.
+It may vary depending on the random seed.
+
+```
+(TaskRunner pid=21297) [response] First, we calculate the number of eggs Janet's ducks lay in a day. Since there are 16 eggs per day and Janet lays these eggs every day, the number of eggs laid in a day is 16.
+(TaskRunner pid=21297)
+(TaskRunner pid=21297) Next, we calculate the number of eggs Janet eats in a day. She eats 3 eggs for breakfast and bakes 4 muffins, so the total number of eggs she eats in a day is 3 + 4 = 7.
+(TaskRunner pid=21297)
+(TaskRunner pid=21297) The number of eggs she sells in a day is the total number of eggs laid minus the number of eggs she eats, which is 16 - 7 = 9 eggs.
+(TaskRunner pid=21297)
+(TaskRunner pid=21297) She sells each egg for $2, so the total amount she makes every day is 9 * 2 = 18 dollars.
+(TaskRunner pid=21297)
+(TaskRunner pid=21297) #### 18
+(TaskRunner pid=21297) #### 18 dollars
+```
+
+# Step 8: Clean up
+
```

Comment:
I note this is only deleting the downloaded yaml. Is there anything else that needs to be removed from the user's environment? Either way, add a sentence to indicate what is/isn't being deleted.

Reply:
The instructions delete all the Kubernetes resources that the audience needs to remove. The Kubernetes cluster and KubeRay operator are likely managed by other teams rather than the audience. That's why I didn't provide instructions to clean them up.